messybrainz/db/data.py

```diff
@@ -219,6 +220,43 @@ def load_recording(connection, messybrainz_id):
     return result
 
 
+def load_recording_from_mbid(connection, musicbrainz_id):
+    """ Return data for a recording with specified MusicBrainz ID.
+
+    Args:
+        connection: sqlalchemy connection to execute db queries with
+        messybrainz_id (uuid): the MusicBrainz ID of the recording
+
+    Returns:
+        dict: the recording data for the recording with specified MessyBrainz ID
+    """
+    query = text("""SELECT rj.data
+                         , d.recording_mbid
+                         , r.artist
+                         , r.release
+                         , r.gid
+                      FROM recording_json rj
+                 LEFT JOIN recording r
+                        ON rj.id = r.data
+                 LEFT JOIN recording_cluster rc
```

Comment:
You shouldn't need to join on recording_cluster -- those tables are leftover from a failed GSoC project we haven't cleaned up. Did you download this messybrainz data? http://data.metabrainz.org/pub/musicbrainz/messybrainz/ If you don't want to do that, (I understand), I could run some queries for you to make sure they work right.

Reply:
Sure I removed the joins to `recording_cluster` and `recording_redirect` from the queries because neither the MBID/MSID functions returned anything that used data from the joins. (I didn't have 12gb for the dump, but please test for me :))